_component_design/tooltip.md

```diff
@@ -311,13 +299,27 @@ Loading states are rarely found when using the tooltip pattern. The loading stat
 
 <section class="static-section" markdown="1">
 
-## Interaction behaviors
+## Help text with a link
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+Add help text to a link when a tooltip provides access to an external webpage. Use the following guidelines adding help text to a link.
+
+- The link should always be external and open in a new tab.
+- Ensure you include an external link icon.
```

Comment:
Is this icon always supposed to appear at the end of the hyperlink text (like in the visual), or does it not matter?